unstract/filesystem/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+[project]
+name = "unstract-filesystem"
+version = "0.0.1"
+description = "Unstract filesystem package"
+authors = [
+    {name = "Zipstack Inc.", email = "devsupport@zipstack.com"},
+]
+dependencies = []
```

Comment:
Please add SDK and DRF as dependencies.
Feel free to specify this library in our [PDM lock automation script](https://github.com/Zipstack/unstract/blob/32b211d7e6eb5aa6a9f15af7fd2e1b94c089ef9d/docker/scripts/pdm-lock-gen/pdm-lock.sh#L70-L82)